src/cascadia/TerminalApp/AboutDialog.cpp

```diff
@@ -90,10 +90,13 @@ namespace winrt::TerminalApp::implementation
             {
                 const auto updates = co_await storeContext.GetAppAndOptionalStorePackageUpdatesAsync();
                 co_await wil::resume_foreground(strongThis->Dispatcher());
-                const auto numUpdates = updates.Size();
-                if (numUpdates > 0)
+                if (updates)
```

Comment:
nit: short circuit allows for `if (updates && updates.Size() > 0)` (which may also just read more svelte-ly